models.py

```diff
@@ -7,6 +7,15 @@
 from dataclasses import dataclass
 
 
+@dataclass
+class PersonalInfo:
+    '''
+    User details
+    '''
+    first_name: str
```

Comment:
nit: you could also have a optional middle_name

Reply:
Sure, would add